ui/app/AppLayouts/Chat/components/LeftTabBottomButtons.qml

```diff
@@ -19,98 +19,96 @@ Column {
         anchors.horizontalCenter: parent.horizontalCenter
     }
 
-    StatusIconTabButton {
+    StatusNavBarTabButton {
         id: walletBtn
         visible: enabled
-        height: enabled ? implicitHeight: 0
+        height: enabled ? implicitHeight : 0
         anchors.horizontalCenter: parent.horizontalCenter
         enabled: isExperimental === "1" || appSettings.isWalletEnabled
         icon.name: "wallet"
         checked: sLayout.currentIndex === Utils.getAppSectionIndex(Constants.wallet)
+        tooltip.text: qsTr("Wallet")
         onClicked: {
             chatsModel.communities.activeCommunity.active = false
             appMain.changeAppSection(Constants.wallet)
         }
     }
 
-    StatusIconTabButton {
+    StatusNavBarTabButton {
         id: browserBtn
         visible: enabled
         height: enabled ? implicitHeight: 0
         anchors.horizontalCenter: parent.horizontalCenter
         enabled: isExperimental === "1" || appSettings.isBrowserEnabled
         icon.name: "browser"
         checked: sLayout.currentIndex === Utils.getAppSectionIndex(Constants.browser)
+        tooltip.text: qsTr("Browser")
         onClicked: {
             chatsModel.communities.activeCommunity.active = false
             appMain.changeAppSection(Constants.browser)
         }
     }
 
-    StatusIconTabButton {
+    StatusNavBarTabButton {
         id: timelineBtn
         visible: enabled
         height: enabled ? implicitHeight: 0
         anchors.horizontalCenter: parent.horizontalCenter
         enabled: isExperimental === "1" || appSettings.timelineEnabled
         icon.name: "status-update"
         checked: sLayout.currentIndex === Utils.getAppSectionIndex(Constants.timeline)
+        tooltip.text: qsTr("Timeline")
         onClicked: {
             chatsModel.communities.activeCommunity.active = false
             appMain.changeAppSection(Constants.timeline)
         }
     }
 
-    StatusIconTabButton {
+    StatusNavBarTabButton {
         id: profileBtn
         visible: enabled
         height: enabled ? implicitHeight: 0
         anchors.horizontalCenter: parent.horizontalCenter
         icon.name: "profile"
         checked: sLayout.currentIndex === Utils.getAppSectionIndex(Constants.profile)
+
+        badge.visible: !profileModel.mnemonic.isBackedUp && sLayout.children[sLayout.currentIndex] !== profileLayoutContainer
+        badge.anchors.rightMargin: 4
+        badge.anchors.topMargin: 5
```

Comment:
Why do we need to adjust the badge anchors for this one?

Reply:
Great question - That's because this particular badge doesn't hold a value and is therefore smaller. It's the little "something is here" indicator on the profile tab button.